### PR TITLE
Partial terragrunt integration

### PR DESCRIPTION
**Pull request by laurit17**
- add .envrc to plugins
- terraform tool integration
- integrating terragrunt fmt

I would also like to integrate terragrunt's validation tool - as mentioned in standup, it requires terraform to be in your `PATH`. However, even with `terraform` in path, I cannot get it to work properly. There is another environment variable, `TERRAGRUNT_TFPATH`, which allows me to get this to work, but we'd have to evaluate that to point to the `$(which terraform)`. So, punting on that integration for now.

### CODE REVIEW

File: linters/terragrunt/plugin.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          cpu:
+            x86_64: amd64
+            arm_64: arm64
+          url: https://github.com/gruntwork-io/terragrunt/releases/download/v${version}/terragrunt_${os}_${cpu}
+        - os: windows
+          cpu: x86_64
+          url: https://github.com/gruntwork-io/terragrunt/releases/download/v${version}/terragrunt_windows_amd64.exe
+  definitions:
+    - name: terragrunt
+      download: terragrunt
```
Comment (TylerJang27) [inline linters/terragrunt/plugin.yaml:39]: Can you add `suggest_if: never`?

File: linters/terragrunt/plugin.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      files: [hcl]
+      environment:
+        - name: PATH
+          list: ["${linter}"]
+      commands:
+        - name: format
+          output: rewrite
+          run: terragrunt hclfmt ${target}
+          success_codes: [0]
+          in_place: true
```
Comment (TylerJang27) [inline linters/terragrunt/plugin.yaml:50]: Can you add `sandbox_type: copy_targets`

File: tools/terraform/plugin.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+version: 0.1
+tools:
+  downloads:
+    - name: terraform
```
Comment (TylerJang27) [inline tools/terraform/plugin.yaml:4]: Is this going to collide with the terraform linter download? A bit worried about cli internals here

File: linters/terragrunt/terragrunt.test.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import { linterFmtTest } from "tests";
+
+linterFmtTest({ linterName: "terragrunt" });
```
Comment (TylerJang27) [inline linters/terragrunt/terragrunt.test.ts:3]: So only the validation subcommand requires terraform to be present? And so we're fine for this subcommand?

**Review by TylerJang27 — COMMENTED** [review]
Some comments that I'd like to have followup discussion, otherwise good

File: linters/terragrunt/terragrunt.test.ts (hunk 4 above)
Comment (laurit17) [reply linters/terragrunt/terragrunt.test.ts:3]: yep, this one doesn't need terraform

File: tools/terraform/plugin.yaml (hunk 3 above)
Comment (laurit17) [reply tools/terraform/plugin.yaml]: fair enough - we won't have a collision, but this should be resolved somehow anyway

**Pull request merged**